vscode-extension/src/util.ts

```diff
@@ -295,3 +295,19 @@ export function urlJoin(...args) {
   return normalize(parts);
 }
 //#endregion
+
+
+/**
+ * AIConfig Vscode extension has a dependency on the Python extension. 
+ * This function retrieves and returns the path to the current python interpreter.
+ * @returns the path to the current python interpreter
+ */
+export async function getPythonPath(): Promise<string> {
+  const pythonExtension = vscode.extensions.getExtension("ms-python.python");
+  if (!pythonExtension.isActive) {
+    await pythonExtension.activate();
```

Comment:
If you don't have the extension installed, will it install as part of activation?

Reply:
It won't... the package.json update now marks that extension as a dependency, so the python extension will be installed when the AIConfig extension is installed

Reply:
+1, updated testplan with some notes